twentytwentyone-blocks/assets/css/style-editor.css

```diff
@@ -12,4 +12,14 @@
 
 .wp-block[data-align="full"] {
 	max-width: inherit;
-}
\ No newline at end of file
+}
+
+/*--------------------------------------------------------------
+# Preformatted Text
+--------------------------------------------------------------*/
+
+pre.wp-block-preformatted {
+	overflow-x: auto;
+	white-space: pre !important;
+	font-size: var(--wp--preset--font-size--extra-small);
```

Comment:
I think this could live in theme.json, once the block supports it. You can see how I did that for the code block: https://github.com/WordPress/gutenberg/pull/27294